Marlin/Configuration.h

```diff
@@ -49,7 +49,7 @@
  *
  * Note: Update also Version.h !
  */
-#define CONFIGURATION_H_VERSION 010100
+#define CONFIGURATION_H_VERSION 010101
```

Comment:
Don't include all these changes in `Configuration.h`. Only the new feature should be added.